sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/listeners/eventdata/MapLoadError.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.maps.plugin.delegates.listeners.eventdata
+
+import com.google.gson.annotations.SerializedName
+
+/** Describes an error while loading the map.  */
+enum class MapLoadError {
+  /** An error while interpreting the contents of the style.  */
+  @SerializedName("style-parse")
```

Comment:
Do we need serialized here?

Reply:
Yes, the data returned from gl-native is in a json format that needs to be serialized to a kotlin compatible format